### ISSUE TITLE
Linux Install Fails

### ISSUE THREAD

**Issue by tekmaven** [chef-partners/azure-chef-extension#179]
New Linux VM, using image `b39f27a8b8c64d52b05eac6a62ebad85__Ubuntu-16_04-LTS-amd64-server-20161130-en-us-30GB` created through Powershell.

Command to enable extension:
```powershell
$config | Set-AzureVMChefExtension -ValidationPem "$Env:USERPROFILE\chef-repo\.chef\company-inc-validator.pem" `
                            -ValidationClientName "company-inc-validator" `
                            -ChefServerUrl "https://api.opscode.com/organizations/company-inc" `
                            -Linux `
                            -AutoUpdateChefClient
```

Chef extension install status:
```json
[
  {
    "status": "NotReady",
    "message": "(000003)Non-zero exit code: 1, enable.sh",
    "code": -1,
    "name": null,
    "operation": null
  }
]
```

`CommandExecution.log`:
```
2016/12/05 21:36:44.435619 INFO [Chef.Bootstrap.WindowsAzure.LinuxChefClient-1210.12.106.1000] Expected handler state: enabled
2016/12/05 21:36:44.454100 INFO [Chef.Bootstrap.WindowsAzure.LinuxChefClient-1210.12.106.1000] Decide which version to use
2016/12/05 21:36:44.693651 INFO [Chef.Bootstrap.WindowsAzure.LinuxChefClient-1210.12.106.1000] Use version: 1210.12.106.1000
2016/12/05 21:36:44.715943 INFO [Chef.Bootstrap.WindowsAzure.LinuxChefClient-1210.12.106.1000] Current handler state is: NotInstalled
2016/12/05 21:36:44.735757 INFO [Chef.Bootstrap.WindowsAzure.LinuxChefClient-1210.12.106.1000] Download extension package
2016/12/05 21:36:47.576114 INFO [Chef.Bootstrap.WindowsAzure.LinuxChefClient-1210.12.106.1000] Unpack extension package
2016/12/05 21:36:47.629032 INFO [Chef.Bootstrap.WindowsAzure.LinuxChefClient-1210.12.106.1000] Initialize extension directory
2016/12/05 21:36:47.648040 INFO [Chef.Bootstrap.WindowsAzure.LinuxChefClient-1210.12.106.1000] Extension has no settings, write empty 0.settings
2016/12/05 21:36:47.676329 INFO [Chef.Bootstrap.WindowsAzure.LinuxChefClient-1210.12.106.1000] Install extension.
2016/12/05 21:36:47.696350 INFO [Chef.Bootstrap.WindowsAzure.LinuxChefClient-1210.12.106.1000] Launch command:install.sh
2016/12/05 21:38:04.817825 INFO [Chef.Bootstrap.WindowsAzure.LinuxChefClient-1210.12.106.1000] Enable extension.
2016/12/05 21:38:04.836376 INFO [Chef.Bootstrap.WindowsAzure.LinuxChefClient-1210.12.106.1000] Launch command:enable.sh
```

**Comment by NimishaS** [comment]
@tekmaven which version of `azure-powershell` are you using?

**Comment by tekmaven** [comment]
1.0.4 😎 Is that my issue?

**Comment by NimishaS** [comment]
I was able to reproduce this issue with 1.x version of `azure-powershell`. But the code of master branch worked successfully for me. Can you try with azure-powershell v2.x?

**Comment by NimishaS** [comment]
@tekmaven , I could reproduce this issue with azure-powershell v3.1.0 too. Since this issue needs fix in `azure-powershell`, opened it here https://github.com/Azure/azure-powershell/issues/3267.
Closing this issue.

**Comment by NimishaS** [comment]
@tekmaven, this issue is resolved in azure-powershell v3.3.0 as mentioned here: https://github.com/Azure/azure-powershell/issues/3267#issuecomment-269528250
